tap_mixpanel/schemas/export.json

```diff
@@ -2,6 +2,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "mp_reserved_insert_id": {
+      "type": ["null", "string"]
+    },
```

Comment:
Do we need this any more?

Reply:
We don't need it, but it doesn't hurt either.

Reply:
how do i dedupe events down stream without this?